README.md

```diff
@@ -14,6 +14,8 @@ website for those who want to contribute.
   MPTCP on a system.
```

Comment:
(Not related to this line)
Please remove the `.DS_Store` file.

(If you use Git from the terminal, I do recommend you to always use `git add -p`: you select what you need, you can easily split commits if needed, and you can do a pre-review. If you use a UI tool, I'm sure there should be an option to select what you want to send ;) )